CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Secondary device attributes escape (`CSI > 0 c`)
 - Support for colon separated SGR 38/48
+- New Ctrl+C binding to cancel search
+- New Ctrl+C binding to cancel search and leave vi mode
```

Comment:
That sounds a bit strange, seems like rebase issue?

Reply:
Correct, let me fix that.